bin/i18n/utils.rb

```diff
@@ -0,0 +1,9 @@
+require 'open3'
+
+def run_bash_script(location)
```

Comment:
plus as a bonus it'll now pipe the output in realtime; no more waiting around without knowing how far along it is.